Makefile

```diff
@@ -0,0 +1,2 @@
+lint:
+	black .
```

Comment:
requrements to be updated later along with other dependencies, needs jupyter-black